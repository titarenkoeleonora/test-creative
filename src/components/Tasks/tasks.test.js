import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter as Router} from "react-router-dom";
import Tasks from "./Tasks";


it('Render Tasks component', () => {
  const tree = renderer
    .create(
      <Router>
        <Tasks />
      </Router>
    );

  expect(tree).toMatchSnapshot();
});