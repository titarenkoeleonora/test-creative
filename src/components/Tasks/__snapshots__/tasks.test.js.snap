// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Tasks component 1`] = `
<section
  className="tasks"
>
  <div
    className="container"
  >
    <div
      className="tasks__wrapper"
    >
      <h2
        className="tasks__title"
      >
        Выполненные задания
      </h2>
      <ul
        className="tasks__list"
      >
        <li
          className="tasks__item"
        >
          <a
            className="tasks__link"
            href="/movement"
            onClick={[Function]}
          >
            Функция сдвига элементов массива вправо на N шагов.
          </a>
        </li>
        <li
          className="tasks__item"
        >
          <a
            className="tasks__link"
            href="/substring"
            onClick={[Function]}
          >
            Функция нахождения самой длинной общей последовательности 2х строк.
          </a>
        </li>
        <li
          className="tasks__item"
        >
          <a
            className="tasks__link"
            href="/checkboxes"
            onClick={[Function]}
          >
            Отображение/скрытие элементов при переключении чекбоксов.
          </a>
        </li>
      </ul>
    </div>
  </div>
</section>
`;
